src/graph.py

```diff
@@ -25,6 +25,8 @@
 from networkx.readwrite import json_graph as nx_json
 import community    # pip install python-louvain
 from sklearn.cluster import SpectralClustering
+from scipy import stats
+from scipy.spatial import distance
```

Comment:
Seems like you aren't using these in the code anymore. Remove them?

Reply:
We're using scipy.stats, removed line 29 though